devservices/config/devservices-credentials.json

```diff
@@ -0,0 +1,5 @@
+{
+  "secret_key": "OxE6Du8quMxWj19f7YDCpIxm6XyU9nWGQJkMWFlkchA",
```

Comment:
Can we generate the credentials for each user individually? I'd like to avoid cases where these keys end up actually being used.

Reply:
I believe this should be possible to generate the credentials when a containerized version of relay is brought up through Sentry. 

Given that this is currently how things are configured in Sentry now (this credentials file seems to be used for all dev setups with relay running), can this be added after this PR goes in?

Reply:
Yep, sounds good! Thanks.